modules/security/rbac/README.md

```diff
@@ -1,99 +1,227 @@
+
+## Cleanup
+
+    For now, lets clean up our Cluster Policy Config so we can carry on with the rest of the lab:
 
-Istio provides every workload with a strong identity - in Kubernetes, the pod's ServiceAccount - which is used to establish mTLS connections between services in the mesh. While establishing mTLS connections, sidecars in the mesh will validate certificates according to [SPIFFE's X.509 SVID spec](https://github.com/spiffe/spiffe/blob/master/standards/X509-SVID.md), which means that after the connection is established we have an authenticated identity of the other party. Istio allows you to write access control policies using those identities to describe which services can communicate.
-
-This config comes in three parts: a cluster wide `ClusterRbacConfig` which sets the mesh's RBAC mode (on, off, inclusive list, exclusive list). We'll start by creating one which requires RBAC for the `default` namespace:
-
-
-1. Require RBAC
-
-    First we'll configure Istio to require RBAC in the mesh:
-    ```sh
-    $ kubectl apply -f modules/security/rbac/config/clusterrbacconfig.yaml
-    ```
-
-    We can see the policy 
-
-    ```yaml
-    $ kubectl describe -n istio-system clusterrbacconfig.rbac.istio.io/default
-
-    apiVersion: "rbac.istio.io/v1alpha1"
-    kind: ClusterRbacConfig
-    metadata:
-      name: default
-    spec:
-      mode: 'ON_WITH_INCLUSION'
-      inclusion:
-        namespaces:
-        - "default"
-    ```
-
-    > The other `mode`s are `ON`, `OFF`, and `ON_WITH_EXCLUSION`
-
-    If we go to the UI, we should see it fail to load with an RBAC related error. We can confirm this by looking at our graphs too.
-
-    ```shell
-    $ curl $INGRESS_IP
-    RBAC: access denied
-    ```
-
-1. Create the `ui-viewer` `Role`
-
-    Now, let's create a `Role` that allows read to use the UI:
-    ```shell
-    $ kubectl apply -f modules/security/rbac/config/ui-viewer.yaml
-    ```
-
-    We can describe the object to see what `Role`s look like in Istio:
     ```shell
-    $ kubectl describe servicerole.rbac.istio.io/ui-viewer
-    apiVersion: "rbac.istio.io/v1alpha1"
-    kind: ServiceRole
-    metadata:
-      name: ui-viewer
-      namespace: default
-    spec:
-      rules:
-      - services: ["ui.default.svc.cluster.local"]
-        methods: ["GET"]
+    kubectl -n hipstershopv1v2 delete AuthorizationPolicy deny-all ingress-policy policy-for-hipstershop redis-policy
     ```
 
-1. Bind the `Role` to Users with a `ServiceRoleBinding`
+## Takeaway
 
-    A `Role` by itself doesn't do us any good, we have to assign the role to ourselves to get the permissions it lists. We'll do this by creating a `ServiceRoleBinding` which lets anyone see the UI:
+Once mTLS is enforced, we can use Istio create RBAC policies to tontrol the traffic in the mesh. While this is powerfull, it may also lead to unattended traffic break. Plane, review and tests your rules before applying them in production.
```

Comment:
```suggestion
Once mTLS is enforced, we can use Istio create Authorization Policies to control the traffic in the mesh. While this is powerful, it may also lead to unintended traffic breaks. Plan, review and tests your rules before applying them in production!
```